hasura/metadata/cron_triggers.yaml

```diff
@@ -1 +1,5 @@
-[]
+- name: check-nominee
+  webhook: '{{HASURA_GRAPHQL_CRON_URL}}/check-nominee'
```

Comment:
We should add a deploy TODO in the PR description to make sure this ENV_VAR is set for staging and prod environemnts too.

Reply:
Added.

Reply:
I think it is better to just write it out? or did hasura generate that itself?
Does hasura automagically create that variable for itself, because it does that for the other `HASURA_GRAPHQL` variables I believe

Reply:
> I think it is better to just write it out? or did hasura generate that itself? Does hasura automagically create that variable for itself, because it does that for the other `HASURA_GRAPHQL` variables I believe

@META-DREAMER recommended the use of env var for webhook handlers, so i advised @jsur to do the same
(https://github.com/coordinape/coordinape/pull/423#discussion_r786488767)

Reply:
Yup, the env var also has a new name: `HASURA_CRON_BASE_URL`.

Reply:
Another way to handle this is to just have a single endpoint to forward all cron events to and then branch the logic out from there. When Hasura calls the event handler it gives all the info on what event was triggered: https://hasura.io/docs/latest/graphql/core/event-triggers/payload.html#json-payload

something like:

```
type EventName = 'check-nominee' | ''
const EventHandlers = {
  'check-nominee': 
}


const handler = (hasuraEventPayload: EventTriggerPayload) => {
    
}
```

Reply:
Makes sense, I can try that approach on the next cron job once this PR is done.